store/postgres/src/entities.rs

```diff
@@ -1357,7 +1379,10 @@ impl Storage {
 /// Delete all entities. This function exists solely for integration tests
 /// and should never be called from any other code. Unfortunately, Rust makes
 /// it very hard to export items just for testing
-pub fn delete_all_entities_for_test_use_only(conn: &PgConnection) -> Result<usize, StoreError> {
+pub fn delete_all_entities_for_test_use_only(
```

Comment:
Want to add `#[cfg(debug_assert)]` since we've been doing that in other test-only code?

Reply:
Yep, added `#[cfg(debug_assertions)]` :grinning:

Reply:
Ah yhea that's what it's called